framework/src/base/Adaptivity.C

```diff
@@ -276,12 +276,9 @@ ErrorVector &
 Adaptivity::getErrorVector(const std::string & indicator_field)
 {
   // Insert or retrieve error vector
-  auto ev_pair_it = _indicator_field_to_error_vector.lower_bound(indicator_field);
-  if (ev_pair_it == _indicator_field_to_error_vector.end() || ev_pair_it->first != indicator_field)
-    ev_pair_it = _indicator_field_to_error_vector.emplace_hint(
-        ev_pair_it, indicator_field, libmesh_make_unique<ErrorVector>());
-
-  return *ev_pair_it->second;
+  auto insert_it = moose_try_emplace(
```

Comment:
It seems weird to name these variables foo_it when they're really pairs and only the first member is an iterator.  I can't think of a good terse alternative though.

Reply:
Oh, you are absolutely right and this is really important to get correct when using auto. `insert_pair` is already better. I'll fix that. It matters when you look at the next line of `.first->second`. Don't you love C++?